backend/cms/views/offers/offer_actions.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required, permission_required
+from django.utils.translation import ugettext as _
+from django.shortcuts import redirect
+
+from ...decorators import region_permission_required
+from ...models import Region, Offer, OfferTemplate
+
+
+@login_required
+@region_permission_required
+@permission_required('cms.manage_offers', raise_exception=True)
+def activate(request, region_slug, offer_template_slug):
+    region = Region.objects.get(slug=region_slug)
+    template = OfferTemplate.objects.get(slug=offer_template_slug)
+    Offer.objects.create(region=region, template=template)
+    messages.success(request, 'Offer "' + template.name + '" ' + _('was successfully activated.'))
```

Comment:
Here, the term "Offer" has to be translated... the ideal solution would be to wrap the whole string into one single translation statement:

```
_('Offer "%(offer_name)s" was successfully activated.') % {'offer_name': template.name}
```

(I didn't test this, but this is the way it should theoretically work - [see docu for details](https://docs.djangoproject.com/en/2.2/topics/i18n/translation/))

Reply:
```suggestion
    messages.success(request, _('Offer "%(offer_name)s" was successfully activated.') % {'offer_name': template.name})
```